RecoTauTag/Configuration/python/boostedHPSPFTaus_cfi.py

```diff
@@ -21,7 +21,7 @@ def addBoostedTaus(process):
     process.ak4PFJetsRecoTauChargedHadronsBoosted.builders[1].dRcone = cms.double(0.3)
     process.ak4PFJetsRecoTauChargedHadronsBoosted.builders[1].dRconeLimitedToJetArea = cms.bool(True)
     process.combinatoricRecoTausBoosted.jetSrc = cms.InputTag('boostedTauSeeds')
-    process.combinatoricRecoTausBoosted.modifiers.remove(process.combinatoricRecoTausBoosted.modifiers[3])
+    #process.combinatoricRecoTausBoosted.modifiers.remove(process.combinatoricRecoTausBoosted.modifiers[3])
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

The following commented out line on L25 is a bit more intuitive but may benefit from a clarification as well

Reply:
I've removed all comment lines. I hope this satisfies all requirements for integration, now.